test/alternator/test_streams.py

```diff
@@ -729,7 +728,6 @@ def test_table_ss_old_image_and_lsi(dynamodb, dynamodbstreams):
     yield table, arn
     table.delete()
 
-@pytest.mark.xfail(reason="Currently fails - see issue #6935, #7030")
```

Comment:
Good, but please also remove the two line comment before the  test_table_ss_old_image_and_lsi() fixture above saying "Currently fails in Alternator because the item's key is missing in OldImage (#6935) and the LSI key is also missing (#7030)." as both of these are already fixed.

Reply:
fixed.